dockerfiles/alpine/Dockerfile.dev

```diff
@@ -0,0 +1,73 @@
+FROM rust:1.72 as builder
```

Comment:
What is the difference between `Dockerfile` and `Dockerfile.dev`?

You can pass specific `BUILD_PROFILE` value to docker build, if you want to get `dev` version